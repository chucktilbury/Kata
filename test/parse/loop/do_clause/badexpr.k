;-------------------------------------------------------------------------------
; Invalid expression is rejected
;-------------------------------------------------------------------------------

func name()() {

    do {} while(dict+1)
}
