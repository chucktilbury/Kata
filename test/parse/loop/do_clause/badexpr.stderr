Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: expected an expression or a ')' but got dict
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: expected a while keyword but got dict
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: expected function body element but got do
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: expected function body but got do
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: expected module item or end of file but got func
Syntax: /home/chuck/Src/Kata/test/parse/loop/do_clause/badexpr.k: 7: 21: module cannot be empty
